tests/garage/envs/mujoco/randomization/test_dynamics_rand.py

```diff
@@ -1,26 +1,296 @@
+import io
 import unittest
 
-from garage.envs.mujoco import SwimmerEnv
+from lxml import etree
+import numpy as np
+
 from garage.envs.mujoco.randomization import Distribution
 from garage.envs.mujoco.randomization import Method
 from garage.envs.mujoco.randomization import randomize
 from garage.envs.mujoco.randomization import Variations
+from garage.envs.mujoco.swimmer_env import SwimmerEnv
 
 
 class TestDynamicsRand(unittest.TestCase):
-    def test_dynamics_rand(self):
+    def setUp(self):
+        self._env = SwimmerEnv()
+        self._bodyname = "mid"
+        self._attribute = "pos"
+
+        # values for UNIFORM Distributions
+        self._lower_bound = np.array([0.5, 0.5, 0.5])
+        self._upper_bound = np.array([1.5, 1.5, 1.5])
+        # values for GAUSSIAN Distributions
+        self._mean = self._lower_bound
+        self._std_dev = self._lower_bound * 2
+
+        # Properties for mocking model generation
+        self._counter = 0
+        self._mocks_abs = [[1.0, 1.0, 1.0], [0.9, 0.87, 0.59], [0.5, 0.5, 0.5],
```

Comment:
If these don't change they can be MOCK_ABS and MOCK_SCALED constants at the top of the file.